packages/ui/src/theme/tokens/components/authenticator.ts

```diff
@@ -1,4 +1,66 @@
-export const authenticator = {
+import {
+  DesignToken,
+  ColorValue,
+  SpaceValue,
+  BorderWidthValue,
+  BorderColorValue,
+  BackgroundColorValue,
+  BorderStyleValue,
+  BoxShadowValue,
+} from '../types/designToken';
+
+interface AuthenticatorModalStyle {
+  width: DesignToken<SpaceValue>;
+  height: DesignToken<SpaceValue>;
+  backgroundColor: DesignToken<ColorValue>;
+  top: DesignToken<SpaceValue>;
+  left: DesignToken<SpaceValue>;
+}
+
+interface AuthenticatorRouterStyle {
+  borderWidth: DesignToken<BorderWidthValue>;
+  borderStyle: DesignToken<BorderStyleValue>;
+  borderColor: DesignToken<BorderColorValue>;
+  backgroundColor: DesignToken<BackgroundColorValue>;
+  boxShadow: DesignToken<BoxShadowValue>;
+}
+
+interface AuthenticatorFooterStyle {
+  paddingBottom: DesignToken<SpaceValue>;
+}
+
+interface AuthenticatorFormStyle {
+  padding: DesignToken<SpaceValue>;
+}
+
+interface AuthenticatorStateStyle {
+  inactive: {
+    backgroundColor: DesignToken<BackgroundColorValue>;
+  };
+}
+
+interface AuthenticatorOrContainerStyle {
+  color: DesignToken<ColorValue>;
+  orLine: {
+    backgroundColor: DesignToken<BackgroundColorValue>;
+  };
+}
+
+interface AuthenticatorContainerToken {
+  widthMax: DesignToken<SpaceValue>;
+}
+export interface AuthenticatorTokens {
```

Comment:
Finding it somewhat difficult to grasp what the structure/contents of a token is meant to be. Looking at the contents of `AuthenticatorTokens` it appears that a token's contents can be:
- a style property (`maxWidth: DesignToken<SpaceValue>`)
- a style object (`form: AuthenticatorFormStyle`)
- another token (`container: AuthenticatorContainerToken`)

Is there some documentation I could look at to better understand this? Is there a semantic difference between a style object (e.g. `AuthenticatorRouterStyle`) versus a design token (e.g. `AuthenticatorContainerToken`)?

Reply:
The `Style` suffix was a mistake, I went with `Tokens` suffix on all the other tokens.